genesyscloud/tfexporter/resource_genesyscloud_tf_export_test.go

```diff
@@ -425,7 +444,14 @@ func TestAccResourceTfExportExcludeFilterResourcesByRegEx(t *testing.T) {
 			exportTestDir,
 			trueValue,
 			[]string{
-				strconv.Quote("genesyscloud_routing_queue::-prod"),
```

Comment:
This was a bug.  We wanted to test that dev|test items were excluded and prod resources were included.  So I needed to make sure the resources created and the resources being checked matched.